code/package.json

```diff
@@ -212,7 +212,7 @@
     "@typescript-eslint/eslint-plugin": "^5.45.0",
     "@typescript-eslint/experimental-utils": "^5.45.0",
     "@typescript-eslint/parser": "^5.45.0",
-    "@vitejs/plugin-react": "^2.1.0",
+    "@vitejs/plugin-react": "^4.0.0",
```

Comment:
Does anybody know why we need this dependency in the first place? And why it's different from the version in the `react-vite` package?

Reply:
It's used to support setups that don't have Vite configs where the React plugin is already loaded, like design systems that don't have apps, and thus doesn't have any default Vite config.